PlayerSkeleton.java

```diff
@@ -19,7 +28,13 @@ public int pickMove(State s, int[][] legalMoves, double[] weights) {
 	* @param s (current state), legalMoves (possible moves for current piece)
 	* @return move with best (highest) heuristic score
 	**/
-	private int tryPossibleMoves(State s, int[][] legalMoves, double[] weights) {
+	private int tryPossibleMoves(State s, int[][] legalMoves, double[] weights, Heuristic h) {
+
+		// By default use new set of features
+		if (h == null) {
```

Comment:
so h is the new heuristics by default. For our final submission we'll indicate our heuristics at which line? Is it on new Line 14? `int best = tryPossibleMoves(s, legalMoves, weights, null);`

Reply:
Please see `TODO` in the original PlayerSkeleton `pickMove` method